params/config.go

```diff
@@ -211,6 +211,9 @@ type WakuConfig struct {
 
 	// SoftBlacklistedPeerIDs is a list of peer ids that should be soft-blacklisted (messages should be dropped but connection kept)
 	SoftBlacklistedPeerIDs []string
+
+	// when true, sends message confirmations
+	EnableConfirmations bool
```

Comment:
A minor thing, go convention is that documentation comments begin with the name of the element they are describing https://blog.golang.org/godoc

```suggestion
	// EnableConfirmations when true, sends message confirmations
	EnableConfirmations bool
```

Reply:
didn't know this, thanks! :)